packages/client/src/components/NavBar/index.js

```diff
@@ -0,0 +1,173 @@
+import React, { useState } from "react";
+import IconButton from '@material-ui/core/IconButton';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Grid from '@material-ui/core/Grid';
+import Link from '@material-ui/core/Link';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import PropTypes from 'prop-types';
+import Modal from '../Modal';
+import Wizard from '../Wizard';
+import Button from '../Button';
+import styled from 'styled-components';
+import { COLORS } from '../../constants';
+import { withStyles } from "@material-ui/styles";
+
+
+const Wiz_content = ['page1', <Button>oh boi</Button>, 'page3']
+
+const TextThemes = {
+  fontSize: {
+    default: 18,
+    lg: 22,
+  },
+  fontWeight: {
+    default: 'normal',
+    lg: 'bold',
+  }
+}
+
+const LinkStyled = styled(Link)`
+  font-family: Muli, sans-serif;
+  font-color: ${COLORS.blue};
+  padding-right: 40px;
+  font-size: ${(props) => TextThemes.fontSize[props.ver]}px;
```

Comment:
we should make sure to handle the case when the `ver` prop supplied doesn't match anything

Reply:
Would setting a default prop value for ver be okay?